c/indexer.c

```diff
@@ -76,16 +76,16 @@ void free_index(struct gzip_index *index)
 
 /* Add an entry to the access point list.  If out of memory, deallocate the
    existing list and return NULL. */
-static struct gzip_index *addpoint(struct gzip_index *index, uint8_t bits,
+static struct gzip_compression_info *addpoint(struct gzip_compression_info *index, uint8_t bits,
```

Comment:
```suggestion
static struct gzip_compression_info *add_checkpoint(struct gzip_compression_info *index, uint8_t bits,
```

Reply:
The variable is still named `index` here and throughout this file. Is changing that in scope?

Reply:
The reason I didn't go there is because
1. I wasn't sure where it would end. Do I also rename the C files?
2. I wasn't sure what to rename it *to*. As I said, "compression_info" is a mouthful. It's one thing in types and function names. As a variable name, it really starts to harm readability.

To be honest, I was kind of hoping that "compression_info" was a bad enough choice that it would spur one of the reviewers here to come up with a better alternative. Seeing none, let me throw this out there: What do you think about **zinfo**? "z" clearly connotes compression, so it's meaning is still pretty clear. Plus, it's fun to say. It's got about the same level of whimsy as "TOC" (which is the other component of the ztoc). And it's short enough that I would feel comfortable using it in file and variable names.

Thoughts 👍 or 👎 ?

Reply:
I think that's ok. If we end up moving towards a `compression` package in go, I would prefer we call it `compression.Info` there instead of `compression.ZInfo`, but zinfo works pretty well as a variable name.

I wish we had a more descriptive name than "info" but I've spent too much time in front of a thesaurus with nothing better to block on that.